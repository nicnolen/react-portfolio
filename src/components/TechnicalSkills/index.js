import React from 'react';

function TechnicalSkills({ name, category, documentation }) {
  return (
    <div>
      {category === 'technical' && (
        <ul>
          <li>{name}</li>
        </ul>
      )}
    </div>
  );
}

export default TechnicalSkills;
