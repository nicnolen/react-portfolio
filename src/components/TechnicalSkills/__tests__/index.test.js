// Import dependencies
import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import TechnicalSkills from '..';

// After each test, make sure there isnt any leftover memory that could give you false results
afterEach(cleanup);

// Declare the components you are testing
describe('TechnicalSkills component', () => {
  // verify the component is rendering properly
  it('TechnicalSkills component renders', () => {
    render(<TechnicalSkills />);
  });

  // compare snapshot versions of the DOM node structure
  it('matches snapshot DOM node structure', () => {
    const { asFragment } = render(<TechnicalSkills />);
    expect(asFragment()).toMatchSnapshot();
  });
});