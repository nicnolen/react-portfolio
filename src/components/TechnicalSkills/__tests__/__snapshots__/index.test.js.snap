// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TechnicalSkills component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <div />
</DocumentFragment>
`;
